khmer/khmer_args.py

```diff
@@ -196,7 +211,11 @@ def info(scriptname, algorithm_list=None):
 
     for alg in algorithm_list:
         sys.stderr.write("||   * ")
-        sys.stderr.write(_algorithms[alg])
+        algstr = _algorithms[alg].encode('utf-8', 'surrogateescape').decode('utf-8', 'replace')
+        try:
+            sys.stderr.write(algstr)
+        except UnicodeEncodeError:
+            sys.stderr.write(algstr.encode(sys.getfilesystemencoding(), 'replace'))
```

Comment:
Weird, but check http://lucumr.pocoo.org/2014/5/12/everything-about-unicode/ to see why this is needed...